rust/operator-binary/src/controller.rs

```diff
@@ -798,11 +798,11 @@ fn build_server_rolegroup_daemonset(
         )])
         .add_env_var_from_field_path("WATCH_NAMESPACE", FieldPathEnvVar::Namespace)
         .add_env_var(
-            "CONSOLE_LOG",
+            "FILE_LOG_LEVEL",
```

Comment:
What should we do about console logging?

To keep behavior as it was, we will need another car added for that.

Reply:
Done in 9156c9c2c9da1c28c815883c127d1405df3b12d7.